client/ayon_core/pipeline/context_tools.py

```diff
@@ -579,3 +586,39 @@ def get_process_id():
     if _process_id is None:
         _process_id = str(uuid.uuid4())
     return _process_id
+
+
+def version_up_current_workfile():
+    """Function to increment and save workfile
+    """
+    host = registered_host()
+    project_name = get_current_project_name()
+    folder_path = get_current_folder_path()
+    task_name = get_current_task_name()
```

Comment:
Should we rely on `host.get_current_context()` here? (which in the end by default does the same, but may differ per host if they wanted to for whatever reason?)

Reply:
These three functions are already done that.
https://github.com/ynput/ayon-core/blob/118b4c189f13741426f4ecabd47c4421ad22325b/client/ayon_core/pipeline/context_tools.py#L347-L372